config/custom-environment-variables.yml

```diff
@@ -40,6 +40,8 @@ public:
 
   rateLimit: 'RATE_LIMIT'
 
+  maxResponseSizeBytes: 'MAX_RESPONSE_SIZE_BYTES'
```

Comment:
After reading this code the effect of this setting was really clear, but before reading the code it wasn't.

How about calling it `fetchLimitBytes`?